config/config.go

```diff
@@ -159,8 +187,8 @@ func BuildFlagArgs(prefix string, ginkgo GinkgoConfigType, reporter DefaultRepor
 		result = append(result, fmt.Sprintf("--%sregexScansFilePath", prefix))
 	}
 
-	if reporter.NoColor {
-		result = append(result, fmt.Sprintf("--%snoColor", prefix))
+	if reporter.ColorizeOutput() {
+		result = append(result, fmt.Sprintf("--%scolor", prefix))
```

Comment:
So do we want to change this to pass `--%snoColor` still? The migration strategy here would be that someone would need to compile their test with an updated `ginkgo` import to get the `--color` flag on their test binary, right?